README.md

```diff
@@ -28,10 +30,35 @@ Create a secret with a private SSH key for accessing the GitHub repository (pull
 neuro secret add gh-rsa @~/.ssh/id_rsa
 ```
 
-Create a secret with your account token for authenticating the training pipeline:
+Create a service account for authenticating the training pipeline:
+
+```shell
+neuro service-account create --name mlops-demo-oss-dogs
+```
+
+Take the full token from the command's output and store it in a secret:
+
+```shell
+neuro secret add platform-config FULL_TOKEN_FROM_OUTPUT
+```
+
+Grant permissions for the new service account
 
 ```shell
-neuro secret add platform-token $(neuro config show-token)
+export USER=<YOUR_USERNAME>
+export PROJECT=mlops-demo-oss-dogs
+export PREFIX="/${USER}/${PROJECT}/"
+export ACCOUNT=${USER}/service-accounts/${PROJECT}
+export ROLE=${USER}/projects/${PROJECT//-/_}
+
+neuro acl grant storage:${PREFIX} ${ROLE} write
+neuro acl grant job:/${ACCOUNT} ${ACCOUNT} manage
+neuro acl grant image:${PREFIX} ${ROLE} read 
+neuro acl grant secret:gh-rsa ${ROLE} read
+neuro acl grant role://${ROLE} ${ACCOUNT} read
```

Comment:
I think we should create a feature request in NF for doing this thing via a dedicated command.
Something like `neuro-flow rbac enable`, or something else. WDYT?

Reply:
As discussed during the standup, I'd rather see it in `neuro-extras`. This set of commands has nothing to do with neuro-flow specifically. The same sharing strategy can be applied to projects wihtout neuro-flow at a..